accelerator/tests/files/test_Gaudi_code.c

```diff
@@ -0,0 +1,21 @@
+// Copyright 2024 Intel Corporation.
```

Comment:
C code is not allowed legally with omnia repo

Reply:
This file is used to verify the tpc compiler works or not, just like test_ROCm_code.cpp. 
Since omnia contains test_ROCm_code.cpp already, does it mean omnia only allows cpp but not c code?

Reply:
We need to remove test_ROCm_code.cpp also

Reply:
@yupengzh-intel, just remove our c code. @priti-parate should take care of the other ones that are not part of this PR.

Reply:
let's remove it and remove the test too